desktop-src/DevNotes/resolvedelayloadsfromdll.md

```diff
@@ -63,19 +63,28 @@ Reserved; must be 0.
 
 ## Return value
 
-The address of the delay-load descriptor, if it is found; otherwise, **NULL**.
+Returns an **NTSTATUS** or error code.
+
+| Return code | Description |
+|--------------|--------------|
+| STATUS_SUCCESS | The operation completed successfully |
+| STATUS_DLL_NOT_FOUND | The Target DLL could not be found |
```

Comment:
It didn't necessarily seem right to enumerate the NTSTATUS values, but Visual Studio's `dloadsup.h` does:
```c
    if (Status == STATUS_DLL_NOT_FOUND) {
        return HRESULT_FROM_WIN32(ERROR_MOD_NOT_FOUND);

    } else {
        return S_OK;
    }
```

That seems like a pretty good indication the only two choices for this function are DLL not found or success.

Reply:
Well, I found another error code option... `STATUS_DELAY_LOAD_FAILED`.  It seems to return that if a delay imported procedure can't be found.